### ISSUE TITLE
new release needed: posting from a split workflow fails with error

### ISSUE THREAD

**Issue by BenWibking** [ZedThree/clang-tidy-review#117]
Posting the review comments from a split workflow fails with the error:
```
Traceback (most recent call last):
Attempting to automatically download review artifacts
  File "/usr/local/bin/post", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/local/lib/python3.[12](https://github.com/quokka-astro/quokka/actions/runs/8300661163/job/22719069742#step:3:13)/site-packages/clang_tidy_review/post.py", line 80, in main
    metadata, review = download_artifacts(pull_request, int(args.workflow_id))
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/clang_tidy_review/__init__.py", line 929, in download_artifacts
    "GET", artifact.archive_download_url, headers=pull.headers("json")
                                                  ^^^^^^^^^^^^
AttributeError: 'PullRequest' object has no attribute 'headers'
```

See https://github.com/quokka-astro/quokka/actions/runs/8300661163/job/22719069742

**Comment by BenWibking** [comment]
It looks like this was fixed in https://github.com/ZedThree/clang-tidy-review/pull/115, but there hasn't been a release yet that incorporates this fix. Can one of the project admins make a new release?

**Comment by ZedThree** [comment]
@BenWibking Apologies, missed that I hadn't made a new release, now available as v0.17.2
